src/components/buttons/buttons.js

```diff
@@ -0,0 +1,37 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import classnames from 'classnames';
+import modifiers from '../../modifiers';
+
+const Buttons = ({
+  children,
+  className,
+  hasAddons,
+  position,
+  ...allProps
+}) => (
+  <div
```

Comment:
Can you change this `div` for the Element component?  also with this there are not need to do the `modifiers.clean` and `modifiers.classnames(allProps)`, because the Element will take care of all the other props. =).

Reply:
Sure, will fix asap.